src/pydocstyle/parser.py

```diff
@@ -665,8 +671,10 @@ def parse_definition(self, class_):
         name = self.current.value
         self.log.debug("parsing %s '%s'", class_.__name__, name)
         self.stream.move()
+        call_names = []
```

Comment:
Should we name this `callable_args` instead?

Reply:
At this point I forgot why I called it this :joy:

`callable_args` it is!